Source/Adobe.Target.Client/Util/Telemetry.cs

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+namespace Adobe.Target.Client.Util
+{
+    using System;
+    using System.Collections.Generic;
+    using Adobe.Target.Delivery.Model;
+
+    internal static class Telemetry
+    {
```

Comment:
We can have these as enhancements in another PR, but can we please avoid adding methods that mutate input parameters. It is confusing and might lead to bugs.

Reply:
Sure, I'll try to fix it in another PR, however we'll need to also fix this in the Java SDK, otherwise getAttributes behaviour will be different from the one in Java SDK, as value merging for duplicate keys will happen differently..